h5p_proplist.go

```diff
@@ -40,6 +44,26 @@ func (p *PropList) Close() error {
 	return p.closeWith(h5pclose)
 }
 
+// Sets the size of the chunks used to store a chunked layout dataset.
+// https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetChunk
+func (p *PropList) SetChunk(ndims uint, dim []uint) error {
```

Comment:
couldn't we drop the `ndims` argument?
AFAICT, we don't need it in Go: slices know their length.

Reply:
Fixed